### PR TITLE
Unbreak syncfree import

### PR DESCRIPTION
**Pull request by yeounoh**
This PR addresses #3224 , until we release a new stable ver >=1.11

### CODE REVIEW

File: test/test_train_mp_imagenet.py [hunk 1]
```diff
@@ -27,18 +27,6 @@
     '--test_only_at_end': {
         'action': 'store_true',
     },
-    # AMP only works with XLA:GPU
```
Comment (JackCaoG) [inline test/test_train_mp_imagenet.py:30]: Let's create a `test_train_mp_imagenet_amp.py` and only run the test if it is on GPU.

File: .circleci/common.sh [hunk 2]
```diff
@@ -56,6 +56,9 @@ function install_deps_pytorch_xla() {
   pip install hypothesis
   pip install cloud-tpu-client
 
+  # Using the Ninja generator requires CMake version 3.13 or greater
+  pip install cmake>=3.13 --upgrade
```
Comment (yeounoh) [inline .circleci/common.sh:60]: cc @miladm FYI

File: test/test_ops.py [hunk 3]
```diff
@@ -252,6 +249,9 @@ def __new__(cls, name, variant_test_name=""):
             # Failing Ops
             # Refer for more info : https://github.com/pytorch/xla/pull/3019#issuecomment-877132385
             # AllowedOpInfoEntry('cdist'),  // precision issue on TPU
+            # AllowedOpInfoEntry('linalg.multi_dot'),  // failing on CPU
```
Comment (yeounoh) [inline test/test_ops.py:252]: #3232 cc @JackCaoG

File: .circleci/common.sh [hunk 4]
```diff
@@ -111,10 +114,19 @@ function run_torch_xla_tests() {
...
+        echo "Running Syncfree Optimizer Test"
+        python test/test_syncfree_optimizers.py
+
+        # Following test scripts are mainly useful for
+        # performance evaluation & comparison among different
+        # amp optimizers.
+        # echo "Running MNIST Test"
+        # python test/test_train_mp_mnist_amp.py --fake_data
+        # echo "Running ImageNet Test"
+        # python etst/test_train_mp_imagenet_amp.py --fake_data
```
Comment (JackCaoG) [inline .circleci/common.sh:128]: mnist is disabled for a different reason, we can run `imagenet_amp` when it is gpu setup

**Review by JackCaoG — APPROVED** [review]
Minor comment, not going to block the merge. You can fix the issue in a separate pr

File: .circleci/common.sh (hunk 4 above)
Comment (JackCaoG) [reply .circleci/common.sh:128]: Make sure you only run 1 epoch through, otherwise it might take too long

Comment (yeounoh) [reply .circleci/common.sh:128]: Created #3233

**Pull request merged**